src/text-editor.coffee

```diff
@@ -1140,7 +1144,23 @@ class TextEditor extends Model
   #   with a positive `groupingInterval` is committed while the previous transaction is
   #   still 'groupable', the two transactions are merged with respect to undo and redo.
   # * `fn` A {Function} to call inside the transaction.
-  transact: (groupingInterval, fn) -> @buffer.transact(groupingInterval, fn)
+  transact: (groupingInterval, fn) ->
+    @batch => @buffer.transact(groupingInterval, fn)
+
+  batch: (fn) ->
```

Comment:
I wonder if we should be more explicit with the name, like `batchPresenterUpdates` or something. Just to clarify what the purpose of this is and to distinguish it from the model-level concept of `transact`.